dashboard/app/controllers/home_controller.rb

```diff
@@ -112,7 +112,9 @@ def init_homepage
       # regular lists of recent scripts.
       exclude_primary_script = true
       @recent_courses = current_user.recent_courses_and_scripts(exclude_primary_script)
-
+      @has_feedback = TeacherFeedback.where(
+        student_id: current_user.id
+      ).count > 0
```

Comment:
nit: can we skip this query if the user is a teacher? 
```
@has_feedback = current_user.student? && TeacherFeedback.where(...
```

Reply:
...or is it theoretically for a teacher in another teacher's section to want to see feedback from their teacher?

Reply:
It is theoretically possible for a teacher in another teacher's section to want to see feedback from their teacher, but we're only showing this component on the student homepage so I think skipping the query for teachers makes sense here.